crates/matrix-sdk/tests/integration/room/common.rs

```diff
@@ -39,7 +39,7 @@ async fn user_presence() {
 
     let _response = client.sync_once(sync_settings).await.unwrap();
 
-    let room = client.get_joined_room(&test_json::DEFAULT_SYNC_ROOM_ID).unwrap();
+    let room = client.get_room(&test_json::DEFAULT_SYNC_ROOM_ID).unwrap();
```

Comment:
I know it's mega annoying to do this after the fact, but: in this entire file, would it make sense to assert that the room is in the join state?

Reply:
Is there much of a point? We have a bunch of tests where the state is asserted.

Reply:
Generally more testing, but I understand it's annoying, so feel free to ignore.